app/src/ui/lib/dark-theme.ts

```diff
@@ -1,12 +1,14 @@
 import { remote } from 'electron'
-import { isMojaveOrLater } from '../../lib/get-os'
+import { isMojaveOrLater, is1809OrLater } from '../../lib/get-os'
 
 export function supportsDarkMode() {
-  if (!__DARWIN__) {
-    return false
+  if (__DARWIN__) {
+    return isMojaveOrLater()
+  } else if (__WIN32__) {
+    return is1809OrLater()
```

Comment:
I'm thinking it could be helpful to leave a comment with context for why this version is required. @say25 you left some context in the issue issue you opened #9015 pointing to [this](https://www.groovypost.com/news/windows-10-build-17666-includes-cloud-clipboard-dark-file-explorer/) article

Reply:
Should I add a note to isMojaveOrLater as well? I tend to agree but I feel like the context in which it is used explains most of it. As we previously discussed as well, I do think its technically possible this would still work on prior versions of Windows 10 but 1809 is old enough (almost 1.5 years) that the feature can just be "attained" by upgrading

Reply:
> I do think its technically possible this would still work on prior versions of Windows 10 but 1809 is old enough (almost 1.5 years) that the feature can just be "attained" by upgrading

☝️ that's exactly the sort of thing I think it could be helpful to capture in a comment :)